Solutions/Marain.Tenancy.Specs/Marain.Tenancy.Specs.csproj

```diff
@@ -29,6 +33,12 @@
     <ProjectReference Include="..\Marain.Tenancy.OpenApi.Service\Marain.Tenancy.OpenApi.Service.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Integration\Features\TenancyClientWithCachingEnabled.feature.cs">
```

Comment:
Do we need this? I thought these relationships were supposed to be detected automatically by SpecFlow now.